Add the doc on Secret customization (#933)
kyma-project/btp-manager: docs/user/03-70-delete-bindings-and-instances.md

## Context

> You can only delete service instances or service bindings created in Kyma using Kyma dashboard or kubectl. You can't perform these operations using the SAP BTP cockpit. To successfully delete a service instance, first delete its service bindings.

Reviewer: do we need the word "successfully" here?

Author: Nope :)

```bash
kubectl delete serviceinstances.services.cloud.sap.com {SERVICE_INSTANCE_NAME}
```
<!-- tabs:end -->

## Next Steps

If you want to delete your Kyma cluster, you may have to first delete your service instances and bindings from it.

> You can't delete your Kyma cluster if any non-deleted service instances in it use the credentials from the SAP Service Manager resources created automatically, as described in [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials). In this case, the existing service instances block the cluster's deletion. Delete your service instances and bindings in Kyma dashboard before you attempt to delete the cluster from the SAP BTP cockpit.

Reviewer: I'd reverse the order of the "if" sentence and of the "before" sentence:

```suggestion
If any non-deleted service instances in your Kyma cluster use the credentials from the SAP Service Manager resources created automatically, you can't delete your Kyma cluster (see [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials). In this case, the existing service instances block the cluster's deletion. Before you attempt to delete the cluster from the SAP BTP cockpit, delete your service instances and bindings in Kyma dashboard.
```

Reviewer: If the link target is the same for both cases, why not extract it and put it into an introductory sentence?

Instead of:
"If you want to delete your Kyma cluster, you may have to first delete your service instances and bindings from it."

Maybe:
"Deleting the service instances and bindings may be a prerequisite for deleting your Kyma cluster. This depends on the way the credentials have been created (for details, see [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials))."

In this case, you can drop the links from the following paragraphs and put the condition first.

Thinking about it more - the content of the Next Steps comes after the reader presumably successfully deleted service instances and bindings (because that is _the_ goal of this document). The scope of _this_ document is not how to delete a Kyma cluster successfully, so are the detailed explanations really necessary here? I think they don't hurt, but where they actually should be (and maybe they are?) is in documentation about deleting a Kyma cluster. (...do we have documentation about deleting a Kyma cluster?).

Author: If you phrase it this way, it will be difficult to understand what exactly [Preconfigured Credentials and Access](https://github.com/kyma-project/btp-manager/pull/933/03-10-preconfigured-secret.md#credentials) explains. And that is the automatic creation of the SAP Service Manager service instance.
The style guide mentions the [exception for very long conditions](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/0e1ae688897b4bc185aa17002bae8ef4.html?state=DRAFT&q=condition#logical-sequence), and I think that's exactly the case.

Author: Let me think about where to put this content. In this section, I only mention what is related to the module. Cluster deletion is a broader topic.

> You can delete your Kyma cluster even if your service instances still exist, provided they all use credentials of SAP Service Manager service instances other than the one created automatically, as described in [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials). In this case, the non-deleted service instances do not block the cluster's deletion. 

Reviewer: ```suggestion
However, if the service instances use credentials of SAP Service Manager service instances other than the one created automatically, you can delete your Kyma cluster even if your service instances still exist (see [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials)). In this case, the non-deleted service instances do not block the cluster's deletion. 
```

Author: Same here.

Author: Skipping "all" and moving the link changes the message.
